ui/components/MesherySettingsEnvButtons.js

```diff
@@ -210,7 +210,7 @@ const MesherySettingsEnvButtons = () => {
           marginRight: '2rem',
         }}
         disabled={!CAN(keys.ADD_CLUSTER.action, keys.ADD_CLUSTER.subject)}
-        data-cy="btnResetDatabase"
+        data-testid="btnResetDatabase"
```

Comment:
looks like we have a mix of camel case and kebab case. Any way we can align those?

Reply:
@ianrwhitney I made these changes. All names are now written in kebab case. :)